js/companiesHandler.js

```diff
@@ -16,7 +16,9 @@ function CompaniesHandler() {
          */
         createElementFor: function(companyData) {
             let companyCard = document.getElementById('company-template').cloneNode(true);
+
             companyCard.removeAttribute('id');
+            companyCard.setAttribute('data-name', companyData);
```

Comment:
For working with data attributes [dataset](https://developer.mozilla.org/en-US/docs/Learn/HTML/Howto/Use_data_attributes) can be used.